### PR TITLE
[Compatible]http v2 and v1 interface compatible

### PR DESCRIPTION
**Pull request by hf200012**
http v2 TableSchemaAction adds the return value of aggregation_type, and modifies the corresponding code of Flink/Spark Connector

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (Yes)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/Field.java [hunk 1]
```diff
@@ -25,16 +25,26 @@
     private String comment;
     private int precision;
     private int scale;
+    private String aggregation_type;
 
     public Field() {
     }
 
-    public Field(String name, String type, String comment, int precision, int scale) {
+    public Field(String name, String type, String comment, int precision, int scale,String aggregation_type) {
```
Comment (morningman) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/Field.java:33]:
```suggestion
    public Field(String name, String type, String comment, int precision, int scale, String aggregation_type) {
```

File: extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java [hunk 2]
```diff
@@ -121,11 +121,11 @@ public void testGetUriStr() throws Exception {
     @Test
     public void testFeResponseToSchema() throws Exception {
         String res = "{\"properties\":[{\"type\":\"TINYINT\",\"name\":\"k1\",\"comment\":\"\"},{\"name\":\"k5\","
-                + "\"scale\":\"0\",\"comment\":\"\",\"type\":\"DECIMALV2\",\"precision\":\"9\"}],\"status\":200}";
+                + "\"scale\":\"0\",\"comment\":\"\",\"type\":\"DECIMALV2\",\"precision\":\"9\",\"}],\"status\":200}";
         Schema expected = new Schema();
         expected.setStatus(200);
-        Field k1 = new Field("k1", "TINYINT", "", 0, 0);
-        Field k5 = new Field("k5", "DECIMALV2", "", 9, 0);
+        Field k1 = new Field("k1", "TINYINT", "", 0, 0,"");
```
Comment (morningman) [inline extension/spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java:127]:
```suggestion
        Field k1 = new Field("k1", "TINYINT", "", 0, 0, "");
```

**Review by morningman — COMMENTED** [review]
Please config your IDEA to enable auto code format

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/Field.java (hunk 1 above)
Comment (hf200012) [reply extension/flink-doris-connector/src/main/java/org/apache/doris/flink/rest/models/Field.java:33]: OK

**Comment by hf200012** [comment]
> Please config your IDEA to enable auto code format

ok

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**